website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,61 @@ curl \
 }
 ```
 
+## Show Data Retention Policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`GET /organizations/:organization_name/relationships/data-retention-policy`
+
+| Parameter            | Description                                                         |
+| ---------------------| --------------------------------------------------------------------|
+| `:organization_name` | The name of the organization to show the data retention policy for. |
+
+This endpoint shows the data retention policy set explicitly on the organization.
+
+When no data retention policy is set on the organization, the site wide data retention policy will be used by default.
+Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
+
+See [Data Retention Policy API](/terraform/cloud-docs/api-docs/data-retention-policies#show-data-retention-policy) for details.
+
+## Create or Update Data Retention Policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`POST /organizations/:organization_name/relationships/data-retention-policy`
+
+| Parameter            | Description                                                                |
+| -------------------- | -------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to update the data retention policy for.      |
+
+This endpoint creates a data retention policy for an organization, or updates the existing policy.
+Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
```

Comment:
Non-blocking nitpick: Should we add an extra line between this line of text and the previous line of text to keep it consistent with the [text on line 948](https://github.com/hashicorp/terraform-docs-common/pull/556/files#diff-f5625a4be7f4b34305dd570e3a8eb331c34e7fd7bc58dab8f5499ed46ceb8737R948)?:
```suggestion

Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
```